Controllers/IncomingSms.cs

```diff
@@ -41,8 +45,8 @@ public IncomingSmsController(ILogger<IncomingSmsController> logger)
     [HttpPost(Name = "PostIncomingSms")]
     public IActionResult Post([FromBody] IncomingSmsWebhookRequest request)
     {
-        string to = "";
-        string from = "";
+        string to = request.from;
```

Comment:
Instead of using request.from, we should have another environment variable for the to variable:

https://github.com/FreeClimbAPI/PHP-SMS-Quickstart/pull/3#discussion_r1585580921